pkgs/tools/networking/openconnect/default.nix

```
@@ -1,56 +1,31 @@
-{ lib
-, stdenv
-, fetchurl
-, pkg-config
-, openssl ? null
-, gnutls ? null
-, gmp
-, libxml2
-, stoken
-, zlib
-, vpnc-scripts
-, PCSC
-, head ? false
-  , fetchFromGitLab
-  , autoreconfHook
-}:
-
-assert (openssl != null) == (gnutls == null);
-
-stdenv.mkDerivation rec {
-  pname = "openconnect${lib.optionalString head "-head"}";
-  version = if head then "2021-05-05" else "8.10";
+{ callPackage, fetchFromGitLab, fetchurl, darwin }:
+let
+  common = opts: callPackage (import ./common.nix opts) {
+    inherit (darwin.apple_sdk.frameworks) PCSC;
+  };
+in rec {
+  openconnect = common rec {
+    version = "8.20";
+    src = fetchurl {
+      url = "ftp://ftp.infradead.org/pub/openconnect/openconnect-${version}.tar.gz";
+      sha256 = "sha256-wUUjhMb3lrruRdTpGa4b/CgdbIiGLh9kaizFE/xE5Ys=";
+    };
+  };
 
-  src =
-    if head then fetchFromGitLab {
+  openconnect_unstable = common rec {
```

```suggestion
  openconnect_unstable = common {
```
Minor nitpick: rec is not needed.

---

Yeah, right, I carried that over from the other attrset :sweat_smile: Thanks for the review!